README.md

```diff
@@ -1,6 +1,6 @@
 # Pluto
 
-[![Build Status](https://dev.azure.com/samueltebbs/pluto/_apis/build/status/SamTebbs33.pluto?branchName=develop)](https://dev.azure.com/samueltebbs/samueltebbs/_build/latest?definitionId=1&branchName=develop)
+[![Build Status](https://github.com/SamTebbs33/pluto/workflows/CI/badge.svg)](https://github.com/SamTebbs33/pluto/actions)
```

Comment:
This file doesn't exist, as it doesn't show up in the read me.

Reply:
The badge only exists for the default branch, which is develop, so it won't show up until we get this into develop :)